docker/test/stress/run.sh

```diff
@@ -322,7 +322,11 @@ else
     clickhouse-client --query="SELECT 'Server version: ', version()"
 
     # Install new package before running stress test because we should use new clickhouse-client and new clickhouse-test
+    # But we should leave old binary in /usr/bin/ for gdb (so it will print sane stacktarces)
+    mv /usr/bin/clickhouse previous_release_package_folder/
     install_packages package_folder
+    mv /usr/bin/clickhouse package_folder/
+    mv previous_release_package_folder/clickhouse /usr/bin/
```

Comment:
FWIW this will `clickhouse-client` from the previous release, while comment above states that new one should be used, however it is doubtful to me, especially after #38717